test/helpers/fixture_deploy_helper.rb

```diff
@@ -0,0 +1,90 @@
+module FixtureDeployHelper
+  # use deploy_fixture_set if you are not adding to or otherwise modifying the template set
+  def deploy_fixture_set(set, subset: nil, wait: true)
```

Comment:
Besides the efficiency aspect, I'd have one concern about that change: `deploy_loaded_fixture_set` assumes that there's ERB in the file (because you could have added some, even if there wasn't any to begin with), so if we always went through it, we'd fail to exercise the script's recognition of pure `.yml` files. Removing that assumption from `deploy_loaded_fixture_set` would require exposing the original extension somewhere in the loaded fixture hash and likely adding a bit of complexity to `deploy_loaded_fixture_set`, so there's a bit of a tradeoff. I could go either way, but I slightly favour the way it is now.